tests/integration/utils2.go

```diff
@@ -22,14 +22,14 @@ import (
 
 	"github.com/bxcodec/faker/support/slice"
```

Comment:
todo: Can you please add some automated testing of encryption at rest?  Perhaps a new test environment variable, and a CI job that runs with it enabled.?

Reply:
Added a new variant for encrypted datastore to the test workflow